pkg/blobserver/google/cloudstorage/storage.go

```diff
@@ -108,16 +111,22 @@ func newFromConfig(_ blobserver.Loader, config jsonconfig.Obj) (blobserver.Stora
 	if dirPrefix != "" && !strings.HasSuffix(dirPrefix, "/") {
 		dirPrefix += "/"
 	}
+
+	dur, err := time.ParseDuration(ratestr)
+	if err != nil {
```

Comment:
You'll probably want to return a more descriptive error here so the user knows that this was from their rate limit.